client/my-sites/site-settings/theme-enhancements.jsx

```diff
@@ -277,7 +279,10 @@ class ThemeEnhancements extends Component {
 	}
 
 	render() {
-		const { siteIsJetpack, translate } = this.props;
+		const { site, siteIsJetpack, translate } = this.props;
+
+		const jetpackVersion = get( site, 'options.jetpack_version', 0 );
```

Comment:
Noting that it's preferred to use the dedicated selector `getSiteOption( state, site.ID, 'jetpack_version' )` vs grabbing the option from the site object.